VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md

```diff
@@ -14,6 +14,7 @@ ms.date: 06/08/2017
 
 Occurs after the workbook is saved.
 
```

Comment:
Hi Tristan.  This is great new content... with a little quibble. We want to reference "internal links" - that is, content elsewhere in this repository - by using relative paths. The OPS topic: https://opsdocs.azurewebsites.net/en-us/opsdocs/partnerdocs/gfm?branch=master#markdown-extensions-supported-by-open-publishing describes the format of an internal link.  Look at the **Links** section of the topic.  There is additional guidance at: https://opsdocs.azurewebsites.net/en-us/opsdocs/partnerdocs/gfm?branch=master#resolve-file-path